sdk/tables/azure-tables/review/tables.api.review.md

```diff
@@ -0,0 +1,695 @@
+    options?: ListEntitiesOptions
+  ): Promise<ListEntitiesResponse<T>>;
+  setAccessPolicy(
+    acl?: SignedIdentifier[],
+    options?: SetAccessPolicyOptions
+  ): Promise<SetAccessPolicyResponse>;
+  readonly tableName: string;
+  updateEntity<T>(
+    entity: Entity<T>,
+    mode?: UpdateMode,
+    eTag?: string,
+    options?: UpdateEntityOptions
+  ): Promise<UpdateEntityResponse>;
+  upsertEntity<T>(
+    entity: Entity<T>,
+    mode?: UpdateMode,
+    options?: UpsertEntityOptions
+  ): Promise<MergeEntityResponse>;
+}
+
+// @public
+export interface TableCreateHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableCreateHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableDeleteEntityHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableDeleteHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableEntityContinuationToken {
+  nextPartitionKey?: string;
+  nextRowKey?: string;
+}
+
+// @public
+export interface TableEntityQueryResponse {
+  odataMetadata?: string;
+  value?: {
+    [propertyName: string]: any;
+  }[];
+}
+
+// @public
+export interface TableGetAccessPolicyHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableInsertEntityHeaders {
+  clientRequestId?: string;
+  contentType?: string;
+  date?: Date;
+  eTag?: string;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableInsertEntityHeaders {
+  clientRequestId?: string;
+  contentType?: string;
+  date?: Date;
+  eTag?: string;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableInsertEntityOptionalParams extends coreHttp.OperationOptions {
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  responsePreference?: ResponseFormat;
+  tableEntityProperties?: {
+    [propertyName: string]: any;
+  };
+  timeout?: number;
+}
+
+// @public
+export interface TableMergeEntityHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  eTag?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableMergeEntityOptionalParams extends coreHttp.OperationOptions {
+  ifMatch?: string;
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  tableEntityProperties?: {
+    [propertyName: string]: any;
+  };
+  timeout?: number;
+}
+
+// @public
+export interface TableQueryEntitiesHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+  xMsContinuationNextPartitionKey?: string;
```

Comment:
`xMsContinuationNextPartitionKey` and `xMsContinuationNextRowKey` should probably be renamed to `nextPartitionKey` and `nextRowKey`